x-pack/legacy/plugins/siem/public/components/navigation/index.test.tsx

```diff
@@ -63,7 +63,7 @@ describe('SIEM Navigation', () => {
     },
     [CONSTANTS.timelineId]: '',
   };
-  const wrapper = shallow(<SiemNavigationComponent {...mockProps} />);
+  const wrapper = mount(<SiemNavigationComponent {...mockProps} />);
```

Comment:
why did we need to move to a `mount`?

Reply:
If I do not mount, I cannot update the props and trigger the `useEffect`

Reply:
https://itnext.io/testing-components-built-using-react-hooks-with-jest-enzyme-edb87d703756